public/js/script.js

```diff
@@ -1,6 +1,7 @@
 let user = -1; // -1 Represents Unselected. 0 Represents Teachers. 1 Represents Student
 const TEACHER = 0;
 const STUDENT = 1;
+var language = "ENGLISH";
```

Comment:
Remove this. The ```language_script.py``` will automatically add a variable called ```LANGUAGE```. Simply pass this variable to GET request.